kernel/signature.mli

```diff
@@ -65,10 +67,17 @@ val add_declaration     : t -> loc -> ident -> staticity -> term -> unit
 (** [add_declaration sg l id st ty] declares the symbol [id] of type [ty]
     and staticity [st] in the environment [sg]. *)
 
+val add_external_declaration : t -> loc -> name -> staticity -> term -> unit
+(** [add_declaration sg l id st ty] declares the symbol [id] of type [ty]
+    and staticity [st] in the environment [sg]. *)
+
 val add_rules           : t -> Rule.rule_infos list -> unit
 (** [add_rules sg rule_lst] adds a list of rule to a symbol in the environement [sg].
     All rules must be on the same symbol. *)
 
+val unsafe : bool ref
+(** If [true] then the type checker cannot be trusted anymore *)
+
```

Comment:
This explanation is a bit too short, I don't recall this flag being discussed. Ideally `signature.mli` should almost be an API documentation.
Maybe describe here the reasons why one would turn this flag to true.

Reply:
I will comment that on #171